packages/material-ui/src/styles/overrides.d.ts

```diff
@@ -124,7 +123,6 @@ export interface ComponentNameToClassKey {
   MuiCircularProgress: CircularProgressClassKey;
   MuiCollapse: CollapseClassKey;
   MuiContainer: ContainerClassKey;
-  MuiCssBaseline: CssBaselineClassKey;
```

Comment:
Yeah the previous implementation is a bit misleading though. We use `ClassKey` for keys that are valid in `classes`. `@global` is not a valid key in `classes` for `CssBaseline`. It just works for `overrides`.

Let's deprecate CssBaselineClassKey and inline the `@global` override.

Reply:
Yeah this was pretty broken. This would be accepted before:

```ts
const theme = createMuiTheme({
  overrides: {
    MuiCssBaseline: {
      "@global": {
        fontWeight: "bold",
        "font-weight": "bold",
        fontSize: 30,
      }
    }
  }
});
```

But that doesn't work at all like you'd think it did.

Reply:
🙃